extensions/dependencies/guppy.html

```diff
@@ -8,3 +8,27 @@
     '/third_party/static/guppy-be6f14/src/transform.xsl',
     '/third_party/static/guppy-be6f14/src/symbols.json');
 </script>
+
+<style>
+    .guppy_active {
+      box-shadow: 0 0 5px rgba(81, 203, 238, 1) !important;
```

Comment:
This if not happening. The style elements that give it a black border instead of the blue one I want to give appear as inline style elements (checked in the inspect tool of Chrome). Even after applying a selector like this - `.conversation-skin-inline-interaction oppia-interactive-math-expression-input .guppy-div.guppy_active` it is not able to overcome the inline style elements.